scripts/oes-data-migration-scripts/migration_v3.12.x_to_v4.0.x.py

```diff
@@ -324,14 +329,27 @@ def get_pi_executions(mysqlcursor_orca):
         logging.error("Exception occurred while getting the pipeline execution : ", exc_info=True)
         raise e
 
+def get_postgres_pi_executions():
```

Comment:
@sudhakaropsmx I think this method (get_postgres_pi_executions()) can be removed and for postgres we can simply call the existing method by passing the postgres cursor as argument ?

Reply:
removed and passing the cursor.